src/components/Button/Button.props.ts

```diff
@@ -129,4 +129,10 @@ export type ButtonProps = {
    * - "link": button link without padding and margin;
    */
   type?: Type,
+
+  /**
+   * Indicates that the linked resource is intended to be downloaded rather than displayed in the browser.
+   * If is a string, the values specifies the default file name for use in labeling the resource in a local file system.
```

Comment:
```suggestion
   * If it is a string, the values specifies the default file name for use in labeling the resource in a local file system.
```

Reply:
fixed in [[d1e589e](https://github.com/mia-platform/design-system/pull/621/commits/d1e589e2a7c23e659ef08dbecb80a9092dbc8701)](https://github.com/mia-platform/design-system/pull/621/commits/d1e589e2a7c23e659ef08dbecb80a9092dbc8701)